web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js

```diff
@@ -43,26 +44,19 @@ export const PredictionsSettings = () => {
       <Elem name={'wrapper'}>
         {loading && <Spinner size={32} />}
 
-        {loaded && versions.length > 0 && (
-          <Description style={{ marginTop: 0, maxWidth: 680 }}>
-            List of predictions available in the project. To learn about how to
-            import predictions,{' '}
-            <a
-              href="https://labelstud.io/guide/predictions.html"
-              target="_blank"
-            >
-              see the documentation
-            </a>
-            .
-          </Description>
-        )}
-
         {loaded && versions.length > 0 && (
           <Elem name={'title-block'}>
             <Elem name={'title'}>Predictions List</Elem>
-            <Caption>
-              Each card is associated with separate model version.
-            </Caption>
+            <Description  style={{ marginTop: 14 }}>
```

Comment:
move this to styles file

Reply:
and why do you description inside `title-block`?

Reply:
ah, that's external component, then this style is good.
but title-block > description is still a little confusing